gamestonk_terminal/fred/fred_api.py

```diff
@@ -0,0 +1,143 @@
+from fredapi import Fred
+import argparse
+import matplotlib.pyplot as plt
+from gamestonk_terminal.helper_funcs import (
+    parse_known_args_and_warn,
+    valid_date,
+    str_to_bool,
+)
+from gamestonk_terminal.config_terminal import API_FRED_KEY
+
+
+def get_GDP(l_args):
+    fred = Fred(api_key=API_FRED_KEY)
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="GDP",
+        description="""
+                GDP
+            """,
+    )
+    parser.add_argument(
+        "-n",
+        dest="n_to_get",
+        type=int,
+        default=-1,
+        required=False,
+        help="Number of GDP Values to Grab",
+    )
+
+    parser.add_argument(
+        "-s",
+        dest="start_date",
+        type=valid_date,
+        default="2015-01-01",
+        required=False,
+        help="Date to Start",
+    )
+
+    parser.add_argument(
+        "-p",
+        dest="plot",
+        type=str_to_bool,
+        default=True,
+        required=False,
+        help="Plot GDP",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, l_args)
+
+        if not ns_parser:
+            return
+
+        gdp = fred.get_series("GDP", ns_parser.start_date)
+
+        if int(ns_parser.n_to_get) > 0:
+            lastn = gdp.tail(int(ns_parser.n_to_get))
+
+        else:
+            lastn = gdp
+
+        for date, val in lastn.iteritems():
+            print(f'Date: {date.strftime("%m-%d-%Y")}, GDP: {val} ')
+
+        if ns_parser.plot:
+            plt.figure()
+            lastn.plot(style="-ok")
+            plt.xlabel("Time")
+            plt.ylabel("GDP")
+            plt.show()
+
```

Comment:
Would be good to keep the same figure style, as in:
<img width="668" alt="Captura de ecrã 2021-03-17, às 12 23 30" src="https://user-images.githubusercontent.com/25267873/111466808-a4949600-871b-11eb-8f1e-79f87f5326bb.png">
to give a better experience.

So from here you can add:
<img width="700" alt="Captura de ecrã 2021-03-17, às 12 20 18" src="https://user-images.githubusercontent.com/25267873/111466531-5384a200-871b-11eb-9c84-160bb70b8647.png">

* Grid
* Title
* Add units on the ylabel